source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
 description: "Instructions on how to secure your Home Assistant installation."
 ---
 
-One major advantage of Home Assistant is that it's not dependent on cloud services. Even if you're only using Home Assistant on a local network, you should take steps to secure your instance.
+One major advantage of Home Assistant is that it is not dependent on cloud services.  By default, you are in control of your data. The decisions you make will affect your security and your privacy.  We take your privacy and security seriously.  Because of this, even if you are only using Home Assistant on a local network, you should take steps to secure your instance.
```

Comment:
I am not sure what the statement `We take your privacy and security seriously` adds to this paragraph, it has nothing to do with the rest of the paragraph which is a long (but fair) way of saying your instance is only as secure as you make it.

Reply:
I agree I don't see the relation in this paragraph with that sentence.

Reply:
This is an elaboration on the original paragraph.  The purpose is to establish a justification for us to give advice.  Additionally it informs the user that we are not providing an inherently insecure system, but it does require work on their part to ensure they maintain their privacy and security which is so dear to us.   

It's the first paragraph on a security recommendations page.  We must establish a posture.  I'll see what I can do to rework or split it.   These are all very important concepts to relay.

Reply:
`We must establish a posture.` with respect, says who? If it really matters, then maybe it deserves a separate paragraph. The value of it does not change the fact that it's awkwardly inserted into the paragraph as is

Reply:
Verbiage modified outside tolerance of original conversation.